crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -974,6 +974,10 @@ unsafe impl<'w> SystemParam for DeferredWorld<'w> {
 
 /// A system local [`SystemParam`].
 ///
+/// A system with a `Local<T>` parameter is given a private value of `T` that persists across system calls.
```

Comment:
I think we should try and merge this with the previous first line to try and make it more useful.